src/components/ListWraper/ListWraper.js

```diff
@@ -1,33 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ListItem from './ListItem/ListItem';
 
-const ListWrapper = ({ items }) => {
-  const { header, users } = items;
+class ListWrapper extends Component {
+ 
+  render(){
+    const { header, users } = this.props.items;
+    
+    return (
+      <table className="table table-striped table-bordered list-wraper">
+        <thead className="table-dark">
+          {header.map(item => {
+            const { id } = item; 
 
-  return (
-    <table className="table table-striped table-bordered list-wraper">
-      <thead className="table-dark">
-        {header.map(item => {
-          const { id } = item; 
+            return (
+              <ListItem key={id} {...item} />
+            )
+          }
+        )}
+        </thead>
+        <tbody>
+        {users.map(item => {
+            const { id } = item; 
 
-          return (
-            <ListItem key={id} {...item} />
-          )
-        }
-      )}
-      </thead>
-      <tbody>
-      {users.map(item => {
-          const { id } = item; 
-
-          return (
-            <ListItem key={id} {...item} />
-          )
-        }
-      )}
-      </tbody>
-    </table>
-  );
+            return (
+              <ListItem key={id} {...item} state={this.props.items} changeState={this.props.changeState} />
```

Comment:
this.props.changeState powinno być zdestrukturyzowane - ale to i tak wyleci bo tak nie mozna